src/storage/txn/scheduler.rs

```diff
@@ -485,6 +534,15 @@ impl Scheduler {
         self.register_report_tick(event_loop);
     }
 
+    /// Event handler for new command.
+    ///
+    /// This method will try to acquire all the necessary latches. If successful, initiates a get
+    /// snapshot for furthur processing; otherwise, adds the command to the waiting queue(s), it
+    /// will be handled later in `wakeup_cmd` when its turn comes.
+    ///
+    /// Note that once a command is ready to execute, the snapshot is always up-to-date during the
+    /// execution because 1) all the conflicting commands (if any) must be in the waiting queues;
+    /// 2) there may be non-conflicitng commands running concurrently, but it doesn't matter.
```

Comment:
The result of Get command depends on the timestamp of the txn who sends the Get command.